addons/binding/org.openhab.binding.chromecast/META-INF/MANIFEST.MF

```diff
@@ -7,7 +7,7 @@ Bundle-Version: 2.1.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ClassPath: .,
  lib/protobuf-java-2.6.0.jar,
- lib/api-v2-0.9.4-SNAPSHOT.jar
+ lib/api-v2-0.10.1-SNAPSHOT.jar
```

Comment:
I assume you have build this from your previous fork - as you did some changes after the review, would it make sense to create a new build from the official master?